spec/models/alaveteli_pro/embargo_extension_spec.rb

```diff
@@ -23,7 +23,7 @@
   end
 
   it "requires an embargo" do
-    embargo_extension.embargo = nil
+    embargo_extension.embargo_id = nil
```

Comment:
Yes, we should have a good understanding of what's happening here before this gets merged.

The [documentation](http://guides.rubyonrails.org/v4.0.13/active_record_validations.html#presence) says that…

> If you want to be sure that an association is present, you'll need to test whether the associated object itself is present, and not the foreign key used to map the association.

That is [not what `EmbargoExtension` is doing](https://github.com/mysociety/alaveteli/blob/develop/app/models/alaveteli_pro/embargo_extension.rb#L14), so I expect that has an effect on this.

You should also check that the behaviour is correct with other similar records that do not have an incorrect validation when the patch is applied. It looks like there are more cases of this incorrect validation use, so I expect they'll need fixing too.